x-pack/plugins/security_solution/public/common/mock/endpoint/http_handler_mock_factory.ts

```diff
@@ -82,7 +82,8 @@ type HttpMethods = keyof Pick<
 const HTTP_METHODS: HttpMethods[] = ['delete', 'fetch', 'get', 'post', 'put', 'head', 'patch'];
 
 export type ApiHandlerMock<R extends ResponseProvidersInterface = ResponseProvidersInterface> = (
-  http: jest.Mocked<HttpStart>
+  http: jest.Mocked<HttpStart>,
+  options?: { ignoreUnMockedApiRouteErrors?: boolean }
```

Comment:
Although I added this here, we probably don't want to use it often. Having the errors come out on the console when working on specific test cases is likely desired so that you are aware of what is not mocked from an API standpoint